### DISCUSSION TITLE
Error on Reset password

### DISCUSSION THREAD

**Discussion by plancked** [postalserver/postal#3206] [Help with using Postal]
Hi,

When trying to reset password, I get:

```
We're sorry, but something went wrong.
If you are the application owner check the logs for more information.
```

Where are the logs?
Thanks
Alex

**Comment by willpower232** [comment]
This usually happens when the smtp settings are missing from the configuration. They are different from the smtp_server settings.

You may be able to see them by running `postal logs web` but I'm half sure that they are written somewhere else.
